resharper/resharper-unity/src/Rider/CodeInsights/UnityCodeInsightsHighlighting.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections.Generic;
+using JetBrains.Annotations;
+using JetBrains.Application.UI.Controls.BulbMenu.Items;
+using JetBrains.DocumentModel;
+using JetBrains.ReSharper.Daemon.CodeInsights;
+using JetBrains.ReSharper.Feature.Services.Daemon;
+using JetBrains.ReSharper.Plugins.Unity.Rider.CodeInsights;
+using JetBrains.ReSharper.Psi;
+using JetBrains.Rider.Model;
+using JetBrains.TextControl.DocumentMarkup;
+using Severity = JetBrains.ReSharper.Feature.Services.Daemon.Severity;
+
+[assembly: RegisterHighlighter(UnityCodeInsightsHighlighting.Id, 
+    EffectType = EffectType.NONE, Layer = HighlighterLayer.SYNTAX + 1, NotRecyclable = true, GroupId = HighlighterGroupIds.HIDDEN)]
```

Comment:
What's the `+1` magic offset? Is there a better layer to use, or is this just what the other code insight highlights do?

Reply:
Just what the other lenses do